Source/Flow/Public/Interfaces/FlowNativeExecutableInterface.h

```diff
@@ -0,0 +1,44 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+#include "Nodes/FlowPin.h"
+
+#include "FlowNativeExecutableInterface.generated.h"
+
+// Implemented only by native (C++) objects that can execute within a flow graph
+// (eg, UFlowNode and UFlowNodeAddOn subclasses implement this)
+UINTERFACE(MinimalAPI, BlueprintType, DisplayName = "Flow Native Executable Interface", meta = (CannotImplementInterfaceInBlueprint))
+class UFlowNativeExecutableInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowNativeExecutableInterface
```

Comment:
These are the flow executable functions that ONLY native flow classes UFlowNode and UFlowNodeAddOn implement.

They bridge the node/addon to the rest of the graph (triggering, and finishing).

Reply:
I have doubts about whether we need this separate interface. It's already included in `UFlowNode` and `UFlowNodeAddOn`, so perhaps it might be moved back to `UFlowNodeBase`?

I might not see a reason for extracting this code to the interface. Is it useful in internal projects?
If there's no important reason for that, it would be worth moving this code back to the node class, as this would simplify understanding code for smaller teams and ease future maintenance.